src/test/java/com/uid2/admin/vertx/SiteServiceTest.java

```diff
@@ -101,6 +103,7 @@ void listSitesHaveSites(Vertx vertx, VertxTestContext testContext) {
                 new Site(11, "site1", false),
                 new Site(12, "site2", true),
                 new Site(13, "site3", false, Set.of("test1.com", "test2.net")),
+                new Site(14, "site4", false, null, Set.of("test1.com", "test2.net"), Set.of("abc", "def")),
```

Comment:
suggestion : change the app_names to be more descriptive for readability eg. 'com.android.app'

Reply:
That makes sense. I created `"com.123.game.app.android", "12345678"`